config.rb

```diff
@@ -5,11 +5,7 @@
 ###
 
 set :markdown_engine, :redcarpet
-set :markdown,
-  fenced_code_blocks: true,
-  smartypants: true,
-  with_toc_data: true,
-  renderer: TechDocsHTMLRenderer
+set :markdown, renderer: TechDocsHTMLRenderer
```

Comment:
So you can. Neat!

We still have to include `Redcarpet::Render::SmartyPants` in the custom renderer class, as it's mixed in [here](https://github.com/middleman/middleman/blob/master/middleman-core/lib/middleman-core/renderers/redcarpet.rb#L30-L35) which is unreachable code with the `renderer` option set.